app/src/main/java/com/money/manager/ex/passcode/PasscodeActivity.java

```diff
@@ -147,10 +146,16 @@ protected void onCreate(Bundle savedInstanceState) {
 		buttonKeyBack.setImageDrawable(ui.getIcon(GoogleMaterial.Icon.gmd_backspace)
             .color(ui.getPrimaryTextColor()));
 
-		// Handle fingerprint authentication
-		findViewById(R.id.fpImageView).setVisibility(View.GONE);
```

Comment:
do you know the reason to produce null here?

Reply:
Not yet... I suppose there is something relete to SDK 34, but still need investigation